scripts/plot_joined_scalars.py

```diff
@@ -0,0 +1,176 @@
+r"""
+Inputs
+-------
+scalars_path : str
+    path to csv with scalar data.
+target : str
+    path where a new directory is created and the plots are saved.
+
+Outputs
+---------
+.png
+    plots in .png format.
+
+Description
+-------------
+The result scalars of all scenarios are plotted in a single plot.
+"""
+import sys
+import os
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+import oemoflex.tools.plots as plots
+from oemoflex.tools.plots import colors_odict
```

Comment:
Please import the labels and colors of oemof_b3 here and pass them to the plotting functions below.

`from oemof_b3 import labels_dict, colors_odict`